caput/containers/__init__.py

```diff
@@ -1,13 +1,62 @@
-"""caput.containers.
+"""A high-level in-memory data container format for :py:mod:`caput.pipeline`.
 
-A high-level in-memory data container format for `caput.pipeline`.
+Containers are built on top of :py:class:`~caput.memdata.BasicCont` and provide a structured way
```

Comment:
Is there a way to get a mention of memdata here.  This renders as

> Containers are built on top of [BasicCont](https://caput--311.org.readthedocs.build/en/311/autoapi/caput/memdata/BasicCont.html#caput.memdata.BasicCont) and provide

The link works fine, but someone who hasn't learned about memdata yet might think `BasicCont` is part of `containers` (given its name).

Maybe:

```suggestion
Containers are built on top of memdata's :py:class:`~caput.memdata.BasicCont` and provide a structured way
```
with a deliberately unlinked "memdata"?